pkg/gitweb

```diff
@@ -0,0 +1,44 @@
+[mirrors]
+http://cdn.kernel.org/pub/software/scm/git/git-2.22.0.tar.xz
+https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.22.0.tar.xz
+
+[vars]
+filesize=5609772
+sha512=75b3dcac36f80281effcd099944de34050a35f3599ce42f86ce60455b6c952039fb0f6438d296e0cc9c0651d4a17f467780dc475669227d3c98ddefe91723d42
+pkgver=18
+desc='git web interface'
+
+[deps]
+musl
+curl
+
+[deps.run]
+perl
+
+[build]
+cd gitweb || exit 1
+
+SRVDIR=/var/www
+REPOS=/home/gitweb
+
+if [ ! -d "$REPOS" ]; then
+	mkdir -p "$REPOS" || exit 1
+fi
+
+make -j$MAKE_THREADS \
+	GITWEB_PROJECTROOT="/home/gitweb" \
+	prefix=/
+
+make gitwebdir=/var/www/gitweb install
+
+if ! grep -q '^www-data:' /etc/passwd ; then
```

Comment:
this is wrong, sabotage packages can be used for crosscompilation in which case it's totally irrelevant what the host has in its /etc. this needs to go into the service script just before `[ "$1" = "--prereqs" ] && exit 0` (prereqs can be used if you deploy your sabotage rootfs to a soon-to-be read-only filesystem. you'd then run each service script with --prereqs so all the thing it needs are in the rootfs)